controllers/library/statistics.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Statistics # :nodoc:
+  include NiceVision
+
+  Counter = ->(arr) { arr.each_with_object(Hash.new(0)) { |el, total| total[el] += 1; } }
+  Sorterator = ->(hash, num) { hash.sort_by { |_, v| v }.reverse.first(num).to_h }
```

Comment:
as far as i understand you can use `hash.sort_by { |_, v| -v }` in order to order the hash descending without reverse